docs/support-bundle-guide.md

```diff
@@ -0,0 +1,255 @@
+| Heap Pprof                  | `agent`, `controller`, `outside`, `Node`, `ExternalNode` | Output of [`pprof.WriteHeapProfile`](https://pkg.go.dev/runtime/pprof#WriteHeapProfile)                                                                                                                                                                               |
+| HNSResources (Windows Only) | `agent`, `outside`, `Node`, `ExternalNode`               | Output of `Get-HNSNetwork` and `Get-HNSEndpoint` commands                                                                                                                                                                                                             |
+| Antrea Agent Info           | `agent`, `outside`, `Node`, `ExternalNode`               | YAML output of `antctl get agentinfo`                                                                                                                                                                                                                                 |
+| Antrea Controller Info      | `controller`, `outside`                                  | YAML output of `antctl get controllerinfo`                                                                                                                                                                                                                            |
+| IP Address Info             | `agent`, `outside`, `Node`, `ExternalNode`               | Output of `ip address` command on Linux or `ipconfig /all` command on Windows                                                                                                                                                                                         |
+| IP Route Info               | `agent`, `outside`, `Node`, `ExternalNode`               | Output of `ip route` on Linux or `route print` on Windows                                                                                                                                                                                                             |
+| IP Link Info                | `agent`, `outside`, `Node`, `ExternalNode`               | Output of `ip link` on Linux or `Get-NetAdapter` on Windows                                                                                                                                                                                                           |
+| Cluster Information         | `outside`                                                | Concatenation of information, including: 1. all Pods, Deployments, Replicasets and Daemonsets in all Namespaces with any resourceVersion. 2. all Nodes with any resourceVersion. 3. all ConfigMaps in all Namespaces with any resourceVersion and label `app=antrea`. |
```

Comment:
"Concatenation of information" -  I do not know this word well. If you are not sure either, consider: "Dump of resources in the cluster".